.gitignore

```diff
@@ -1,12 +1,9 @@
 .nextflow*
 work/
-/data/
+data/
```

Comment:
Why did you make this change? Since this means that any subdirectory `data` is ignored, and we do have a `tests/data` directory that we are maintaining.

Reply:
nf-core lint requires this file to match. I can revert, but then I'd have to add an another ignore to the nf-core linting, which is also fine.

Reply:
Reverted in https://github.com/phac-nml/fetchdatairidanext/commit/1a5899aaf611c57b54cb54175f45eb27259d78b3